bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java

```diff
@@ -11,10 +11,15 @@
  */
 public interface IBootstrapSettings {
 
+    /**
+     * The version of Bootstrap for Webjars
+     */
+    String BOOTSTRAP_WEBJARS_VERSION = "4.6.0-1";
+    
     /**
      * The version of Bootstrap
      */
-    String VERSION = "4.6.0-1";
+    String BOOTSTRAP_VERSION = "4.6.0";
```

Comment:
Please revert the constant renaming.

Reply:
Why do we have now two versions - plain and webjars ?

Reply:
There was a fix release "4.6.0-1" for the bootstrap 4.6.0 webjar. The  BOOTSTRAP_WEBJARS_VERSION constant is used in the BootstrapJsReferenceTest